dashboard/app/views/levels/editors/_applab.html.haml

```diff
@@ -85,20 +81,7 @@
   :javascript
     levelbuilder.initializeCodeMirror('level_data_tables', 'javascript');
 
-.field
-  = f.label 'Key-Value Data'
-  You can copy the data array from the debug view in the Data tab in App Lab.
-  %pre
-    = preserve do
-      :escaped
-        Example:
-        {
-          "click_count": 5,
-          "button_color": "blue"
-        }
-  = f.text_area 'data_properties', placeholder: 'Insert JSON Key-Value Data', rows: 4, value: @level.data_properties or ''
-  :javascript
-    levelbuilder.initializeCodeMirror('level_data_properties', 'javascript');
+= render partial: 'levels/editors/key_value_data', locals: {f: f}
```

Comment:
nice extractions! I think these are good to merge, but as a follow-up it could help to have a separate folder (maybe `levels/editors/fields`) for partials like this which represent a single editable field, or any other organizational structure you prefer to add some structure to the growing number of partials in this directory.

Reply:
Added this on https://github.com/code-dot-org/code-dot-org/pull/32116